ubuntudesign/www.ubuntu.com
# Tidy up core and remove YUI dep

## Done
- Removed redundent JS
- Converted from YUI to ES5
- Tidied up the functions

## QA
- Check the menu works normally on large screen
- Scale down to small screens and check the toggles for search and menu work the same as live
- Check breadcrumbs conceal and reveal correctly
- Check there are no js errors
